frontend/playground-frontend/src/features/Feedback/redux/feedbackApi.ts

```diff
@@ -0,0 +1,25 @@
+import { backendApi } from "@/common/redux/backendApi";
+import { auth } from "@/common/utils/firebase";
+
+const feedbackApi = backendApi.injectEndpoints({
+  endpoints: (builder) => ({
+    sendFeedbackData: builder.query({
+      query: (args) => {
+        const { email_address, subject, body_text } = args;
+        return {
+        url: "/api/aws/sendEmail",
+        method: "POST",
+        body: {
+          user: auth.currentUser,
+          email_address: email_address,
+          subject: subject,
+          body_text: body_text,
+        },
+      }
+      },
+    }),
+  }),
+  overrideExisting: true
```

Comment:
@dwu359 can you confirm if this is needed?

@farisdurrani do you by any chance know abt this as well?

Reply:
override existing is not strictly needed here. All it does is override the same endpoint name if it exists (which it doesn't). Keeping it or removing it does not affect anything